### ISSUE TITLE
Add support for .jpg format in `preview_plot()`

### ISSUE THREAD

**Issue by kadyb** [brazil-data-cube/rstac#161]
Currently the `preview_plot()` function doesn't support the .jpg format, but adding it should be rather simple?

```r
url = "https://e84-earth-search-sentinel-data.s3.us-west-2.amazonaws.com/sentinel-2-c1-l2a/34/U/EB/2023/11/S2A_T34UEB_20231107T093339_L2A/L2A_PVI.jpg"
rstac::preview_plot(url)
#> Error: File type 'jpg' not supported
```

```r
# .jpg support 
url = "https://e84-earth-search-sentinel-data.s3.us-west-2.amazonaws.com/sentinel-2-c1-l2a/34/U/EB/2023/11/S2A_T34UEB_20231107T093339_L2A/L2A_PVI.jpg"
tmp = tempfile(fileext = ".jpg")
download.file(url, tmp, mode = "wb")
img = jpeg::readJPEG(tmp)
plot(1:10, type = "n", xlab = NA, ylab = NA, axes = FALSE)
grid::grid.raster(img)
```

**Comment by kadyb** [comment]
@rolfsimoes, what is the reason for reverting this change? Is something wrong?

**Comment by rolfsimoes** [comment]
Sorry, I made a mistake merging it to the `master` branch, so I had to revert. 
The PR now is merged into branch `b-1.0.1`. 

We just use `master` after release to `CRAN`. You can install this version by using `remotes` package:

```r
remotes::install_github("brazil-data-cube/rstac@b-1.0.1")
```

After gather all updates I'll release it to the `master` branch.
